winui/Kanban/events.md

```diff
@@ -0,0 +1,762 @@
+---
+layout: post
+title: Events in WinUI Kanban control | Syncfusion
+description: Learn here all about the interaction-related events support in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Events in WinUI Kanban Control
+
+## Notifying Kanban card drag start
+
+The `CardDragStarting` event occurs when the card start to drag. We can get the below details from the `CardDragStarting` event.
```

Comment:
The `CardDragStarting` event occurs when a card begins to be dragged. We can get the below details from the `CardDragStarting` event.

Reply:
Content changed